src/sif/autocost.cc

```diff
@@ -66,6 +66,9 @@ constexpr float kTaxiFactor = 0.85f;
 // Do not avoid alleys by default
 constexpr float kDefaultAlleyFactor = 1.0f;
 
+// Avoid living street by default
+constexpr float kDefaultLivingStreetFactor = 3.0f;
```

Comment:
Need to understand, what should be used as a default value. 3.0 solves Barcelona cases from the issue